pyquil/operator_estimation.py

```diff
@@ -27,6 +28,14 @@
 log = logging.getLogger(__name__)
 
 
+class SymmType(IntEnum):
```

Comment:
let's call it `SymmetrizationLevel` or something a bit more verbose & descriptive -- this pretty much just for the source